CenterForOpenScience/ember-osf-web
# [ENG-3518] Sunset quickfiles

-   Ticket: [ENG-3518](https://openscience.atlassian.net/browse/ENG-3518)
-   Feature flag: n/a

## Purpose

Remove quickfiles from the front end for the sunsetting of Quickfiles.

## Summary of Changes

1. Remove guid-user files
2. Remove quickfiles link
3. Remove quickfiles from OSF Home navbar
4. Delete data from guid-file files (keeping the files because we'll use them for Files Page Redesign)
5. Link to user profile as href not as route (we don't host the profile page, so it shouldn't be routing in-app)
6. Remove links between files and user models
7. Remove quickfiles route and template
8. Remove file-browser and file-browser-items
9. Remove file-share-button component
10. Remove quickfiles-nav component
11. Remove quickfiles from mirage
12. Remove ember-click-outside from package.json
13. Remove and fix various tests
14. Remove quickfiles-related translation keys
15. Update yarn.lock

## Screenshot(s)

<img width="1061" alt="Screen Shot 2022-01-21 at 10 17 04 AM" src="https://user-images.githubusercontent.com/6599111/150551899-0466f2bc-094c-45f1-8188-40ffc23a3228.png">


## Side Effects

It's possible, but I don't think I could have removed anything being used by anything else without noticing and causing the build or tests to fail. 

## QA Notes

Make sure quickfiles is really gone from ember-osf-web. Make sure other file operations, like on draft registrations, still works properly. Check that links to user profiles still work properly (contributor management, institutional dashboard).
